app.py

```diff
@@ -4,13 +4,29 @@
 #create an instance of Flask class
 app = Flask(__name__)
 
-@app.route('/')
+@app.route('/lynn')
+def login():
+    return render_template('login.html')
+
+@app.route('/lynn')
 def index():
     return render_template('index.html')
 
-@app.route('/user/<name>')
-def user(name):
-    return render_template('user.html', name=name)
+@app.route('/lynn')
+def lynn():
+    return render_template('lynn.html')
+
+@app.route('/lynn')
+def lynn():
+    return render_template('friends.html')
+
+@app.route('/lynn')
+def lynn():
+    return render_template('helpcenter.html')
+
+@app.route('/lynn')
```

Comment:
Same goes here.

Reply:
Also, we must never remove the root route ie ('/') . That is the base of your application.